packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security-rbac.cy.ts

```diff
@@ -0,0 +1,69 @@
+describe("TESTS: Access secured by Elytron OIDC client with RBAC", () => {
+  let wildfly: string;
+  let ssoServer: string;
+  const wildflyPortOffset = 1;
+  const profileInfo =
+    "#hal-header-container > div.collapse.navbar-collapse.hal-header-collapse > ul.nav.navbar-nav.navbar-utility > li:nth-child(5) > a";
+  const errorMessage =
```

Comment:
Is this a normal pop-up error message? If it's the case, I think there is a function for checking success pop-up messages where you can get inspired.

Reply:
https://github.com/hal/berg/blob/main/packages/testsuite/cypress/support/verification-utils.ts#L3

Reply:
https://github.com/hal/berg/blob/main/packages/testsuite/cypress/support/form-editing.ts#L26